sdk/export/trace/trace.go

```diff
@@ -40,16 +40,20 @@ type SpanExporter interface {
 	// calls this function will not implement any retry logic. All errors
 	// returned by this function are considered unrecoverable and will be
 	// reported to a configured error Handler.
-	ExportSpans(ctx context.Context, spanData []*SpanData) error
+	ExportSpans(ctx context.Context, ss []*SpanSnapshot) error
```

Comment:
I'll resolve this since this is discussed in https://github.com/open-telemetry/opentelemetry-go/issues/1380 and in any case can't be done here due to the import cycle problem.